### PR TITLE
Style action bars. Fixes #369

### PR DESCRIPTION
**Pull request by jfrankl**

### CODE REVIEW

**Comment by jfrankl** [comment]
image

File: src/nyc_trees/apps/survey/templates/survey/partials/reserved_blockface_popup.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h4 class="modal-title" id="reservation-modal-label">Cancel Reservation</h4>
                 <p>Are you sure you want to cancel?</p>
             </div>
             <div class="modal-footer">
-                <button type="button" class="btn btn-default pull-left" data-dismiss="modal">Keep Reservation</button>
+                <button type="button" class="btn btn-plain pull-left" data-dismiss="modal">No</button>
```
Comment (maurizi) [inline src/nyc_trees/apps/survey/templates/survey/partials/reserved_blockface_popup.html]: When I hover over this button, the text turns the same color as the background, making it "invisible"

**Comment by maurizi** [comment]
Other than my one comment, +1, works well.

**Comment by jfrankl** [comment]
I am fixing that comment as part of #602 so I am going to merge this now.

**Pull request merged**